packages/react-ai/src/index.ts

```diff
@@ -1,7 +1,5 @@
-import { createAIConversation } from './components/AIConversation';
-import { createAIHooks } from './hooks/createAIHooks';
-import { AIContextProvider } from './hooks/AIContextProvider';
-import { AIConversation } from './components/AIConversation';
+import { createAIConversation, AIConversation } from './components';
+import { createAIHooks, AIContextProvider } from './hooks';
```

Comment:
Dont' know where to comment on this, but can we please add an exports test like we have in the react package? https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/__tests__/exports.ts